src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -451,6 +452,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
   public componentOptionsModel: ComponentOptionsModel;
   public usageAnalytics: IAnalyticsClient;
   public historyManager: IHistoryManager;
+  public ariaLive: AriaLive;
```

Comment:
Does it need to be public / a property on the search interface ?

I don't think someone doing an integration would need to interact with it, or that multiple component need to access it to perform their task (eg: usageAnalytics/queryController, etc.).

Reply:
Indeed, it is not being used anywhere atm. I was thinking in the future, it may be helpful to have it be `public` if different components want to announce changes. e.g. "Results are now in card mode", "Results are now sorted by date descending". For now, I can change to private.

Edit: Since ariaLive is not used in the file, I removed the property and removed the unit test that checks it's instantiation.